app/src/main/java/com/myvetpath/myvetpath/Picture.java

```diff
@@ -14,19 +16,25 @@
     public static final String COLUMN_DATETAKEN = "DateTaken";
 
     //Create Table String
-    public static final String CREATE_TABLE = "CREATE TABLE " + TABLE_NAME + "(" + COLUMN_ID +
-            "INTEGER PRIMARY KEY," + COLUMN_IMAGETITLE + "TEXT )";
+    public static final String CREATE_TABLE = "CREATE TABLE " + TABLE_NAME + "("
+            + COLUMN_ID + "INTEGER PRIMARY KEY,"
+            + COLUMN_IMAGETITLE + "TEXT"
+            + COLUMN_INTERNAL + "INTEGER"
+            + COLUMN_IMAGELINK + "TEXT" //will have to double check how this works
+            + COLUMN_LATITUDE + "BIGINT"
+            + COLUMN_LONGITUDE + "BIGINT" //forgot what we where doing with these.
+            + COLUMN_DATETAKEN + "BIGINT"
+            + ")";
 
 
     //Object Fields
-    //Commented out variables in order to reduce the amount of testing needed.
     private int imageID;
     private int internalID;
-    //private Bitmap imageLink; May be able to use the object image?
+    private Bitmap imageLink;
```

Comment:
I remember you bringing up using a Blob type for the image. So we're going with Bitmap? Just wanted to make sure.

Reply:
Based on our talk from earlier, it looks like we're going to want a string instead of a bitmap here.